### PR TITLE
updates to favoriting

### LINKED ISSUE #10266: Update 'Follow' to 'Favorites', 'Add to Favorites'
- Change label to Favorite in profile menu
- Change filter labels in explorer page 
- Add purple alert banner when star icon is clicked in grant explorer
- Add purple alert banner when star icon is clicked in grant detail page 

Figma file: [Follow -> Favorite](https://www.figma.com/file/8QjqOzDBd4HiE37YqcocK7/GR13?node-id=308%3A4891)

### PR DESCRIPTION
**Pull request by chibie**
##### Description

* update verbiage
* add an alert on adding favorite

##### Refers/Fixes

closes #10266 

##### Testing

https://share.vidyard.com/watch/3XwVC7mbTZ9GqW4LpVEKLF?

### CODE REVIEW

File: app/assets/v2/js/grants/_detail-component.js [hunk 1]
```diff
@@ -526,6 +526,7 @@ Vue.component('grant-details', {
       logo: null,
       logoPreview: null,
       logoBackground: null,
+      dismissFavoriteAlert: 0,
```
[the remark this answers is not in the document]
Comment (chibie) [reply app/assets/v2/js/grants/_detail-component.js:529]: it's set to 10 for a 10seconds countdown in https://github.com/gitcoinco/web/blob/0650b728b33c57d321a26cb1dabf04c295a1d007/app/grants/templates/grants/components/card.html#L21

[the remark this answers is not in the document]
Comment (chibie) [reply app/assets/v2/js/grants/_detail-component.js:530]: done https://github.com/gitcoinco/web/pull/10344/commits/3b6c0ca8d6eef57bf81fd731db174f2275a08868

[the remark this answers is not in the document]
Comment (chibie) [reply app/assets/v2/js/grants/_detail-component.js]: done https://github.com/gitcoinco/web/pull/10344/commits/3a96f2dfc9001e9809d5274b1b9738f09e3772ee

**Review by thelostone-mc — APPROVED** [review]

File: app/grants/templates/grants/explorer.html [hunk 2]
```diff
@@ -55,6 +55,15 @@
...
 
+    .alert-fixed-top {
+      position: fixed;
+      top: 80px;
+      right: 0;
+      z-index: 1030;
+      width: 450px;
+      max-width: 95%;
+    }
+
```
Comment (gravityblast) [inline app/grants/templates/grants/explorer.html:66]: would it make sense to share this style in a different css given that it's used in the other page as well?

Comment (gdixon) [reply app/grants/templates/grants/explorer.html:66]: Yeah, this could live in `app/assets/v2/scss/gc-utilities.scss` @chibie

Comment (chibie) [reply app/grants/templates/grants/explorer.html]: done ✅  https://github.com/gitcoinco/web/pull/10344/commits/7b242bb5726a03ac3ad3ddc87442d94326b8e563

**Pull request merged**